modules/sdk-coin-ada/src/lib/transaction.ts

```diff
@@ -140,6 +146,7 @@ export class Transaction extends BaseTransaction {
       result.outputs.push({
         address: output.address().to_bech32(),
         amount: output.amount().coin().to_str(),
+        multiAssets: output.amount().multiasset(),
```

Comment:
What happens when the multi asset doesn't exist on the output?

Reply:
missed it; will add an OR condition here